packages/mdc-theme/_variables.scss

```diff
@@ -42,11 +42,9 @@ $mdc-theme-background: #fff !default; // White
 $mdc-theme-primary-tone: mdc-theme-light-or-dark($mdc-theme-primary);
 $mdc-theme-primary-light-tone: mdc-theme-light-or-dark($mdc-theme-primary-light);
 $mdc-theme-primary-dark-tone: mdc-theme-light-or-dark($mdc-theme-primary-dark);
-
```

Comment:
The first 4 changes to this file (removed newlines, added trailing commas) seem like they can be reverted?